src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/New-AzureRmRecoveryServicesAsrPolicy.md

```diff
@@ -37,6 +37,13 @@ New-AzureRmRecoveryServicesAsrPolicy [-AzureToVMware] -Name <String> -RecoveryPo
  [<CommonParameters>]
 ```
 
+### AzureToAzure
```

Comment:
Please add an example for this new parameter set

Reply:
sure.